QuizApp/App/Screens/Quiz/QuizSectionHeaderView.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+final class QuizSectionHeaderView: UICollectionReusableView {
+
+    static let reuseIdentifier = String(describing: QuizSectionHeaderView.self)
+
+    var titleLabel: UILabel!
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+}
+
+extension QuizSectionHeaderView {
+
+    func set(for category: Category) {
+        titleLabel.text = category.rawValue
+    }
```

Comment:
I would personally keep this in the original class definition, not extension. Extensions are great, but should be used either for extracting functions related to a specific functionality or a protocol. Overdoing them reduces readability. 

A setter like this one is something that should be visible right away.